crawler/processing/update_sample_exporter.py

```diff
@@ -0,0 +1,250 @@
+import copy
```

Comment:
Maybe a bit more top level documentation on what is going on?

Reply:
Yes that's entirely fair.  Will do that.